pubspec.yaml

```diff
@@ -11,5 +11,5 @@ dependencies:
   nanoid: ^1.0.0
 
 dev_dependencies:
-  dartclub_lint: ^1.0.0
+  # dartclub_lint: ^1.0.0
```

Comment:
There should definitely be another linter instead of just deactivating the old one
Maybe use dart_lints package